### PR TITLE
[FIX] Flathub release workflow

### PR DESCRIPTION
**Pull request by flavioislima**
Fix the current issues:
- Workflow failed to add release notes and open the PR because the structure of the original XML changed due to Flathub made it mandatory to add a Copyright comment at the top. So I adjusted the position of the `ul` element to the correct one.
- Moved from xml2js over to fast-XML-parse that is the most updated lib and doesn't mess with formatting.

Tested it running the command locally and the result can be observed on this PR:
https://github.com/flathub/xyz.hyperplay.HyperPlay/pull/80

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by eliobricenov — APPROVED** [review]

**Comment by flavioislima** [comment]
Rewrote the code to use a different library because the previous flathub PR was failing. Now it passed and I merged it, everything works fine.

File: flathub/update-flathub.ts [hunk 1]
```diff
@@ -101,43 +100,69 @@ async function main() {
...
-  hpXml = convert.js2xml(hpXmlJson)
-  fs.writeFileSync(xmlFilePath, hpXml)
+
+  releaseListTag.ul = [...releaseNotesElements] // Set the <ul> element to the generated <li> elements
+
+  console.log('new releaseListTag = ', JSON.stringify(releaseListTag, null, 2))
+
+  const updatedHpXml = builder.build(hpXmlJson)
+
+  console.log('updatedHpXml = ', updatedHpXml)
```
Comment (BrettCleary) [inline flathub/update-flathub.ts:163]: just double checking if you meant to leave the console logs in

**Review by BrettCleary — APPROVED** [review]
doing God's work sir 🙏

File: flathub/update-flathub.ts (hunk 1 above)
Comment (flavioislima) [reply flathub/update-flathub.ts:163]: Yes, good to debug in case we have future issues like the previous one.
Also causes no harm 😬

**Pull request merged**